### PR TITLE
[stats] Default enable include histograms api for SinkPredicates

### PR DESCRIPTION
**Pull request by pradeepcrao**
Commit Message:
Flip the runtime flag `envoy.reloadable_features.enable_include_histograms` to be true by default.
Additional Description:
Risk Level: Low
Testing: Existing tests cover both states of flag
Docs Changes: N/A
Release Notes: Added
Platform Specific Features: N/A

### CODE REVIEW

**Review by jmarantz — COMMENTED** [review]
have we already done all the internal changes to include histograms we want?

**Comment by jmarantz** [comment]
also needs main merge

File: source/common/runtime/runtime_features.cc [hunk 1]
```diff
@@ -93,6 +93,7 @@ RUNTIME_GUARD(envoy_reloadable_features_validate_grpc_header_before_log_grpc_sta
 RUNTIME_GUARD(envoy_reloadable_features_validate_upstream_headers);
 RUNTIME_GUARD(envoy_restart_features_send_goaway_for_premature_rst_streams);
 RUNTIME_GUARD(envoy_restart_features_udp_read_normalize_addresses);
+RUNTIME_GUARD(envoy_reloadable_features_enable_include_histograms);
```
Comment (botengyao) [inline source/common/runtime/runtime_features.cc:96]: Is there any test to cover this? Looks like we only flip the flag?

**Comment by jmarantz** [comment]
@pradeepcrao should we re-open this?

**Comment by pradeepcrao** [comment]
> @pradeepcrao should we re-open this?

Hi @jmarantz, is this good to go now?

**Comment by jmarantz** [comment]
needs format fix

Also boteng asked about testing above

/wait

**Comment by pradeepcrao** [comment]
> needs format fix
> 
> Also boteng asked about testing above
> 
> /wait

Fixed format (hopefully).

We do parameterised testing with the runtime flag off and on here: https://github.com/envoyproxy/envoy/blob/main/test/common/stats/thread_local_store_test.cc#L2148 

Is that sufficient?

Thanks,
Pradeep

**Comment by jmarantz** [comment]
@botengyao can you review in the context of your testing comment above?

**Review by botengyao — APPROVED** [review]
Thanks for the clean up!

**Review by jmarantz — APPROVED** [review]
@ggreenway for senior approval.

**Review by ggreenway — APPROVED** [review]

**Pull request merged**